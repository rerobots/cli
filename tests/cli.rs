// SCL <scott@rerobots.net>
// Copyright (C) 2021 rerobots, Inc.

use assert_cmd::Command;


#[test]
fn prints_version() {
    let mut cmd = Command::cargo_bin("rerobots").unwrap();
    let assert = cmd.arg("-V").assert();
    assert
        .stdout(format!("{}\n", env!("CARGO_PKG_VERSION")))
        .success();
}
