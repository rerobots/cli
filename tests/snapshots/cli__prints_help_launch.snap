---
source: tests/cli.rs
assertion_line: 71
expression: "String::from_utf8(output.stdout).unwrap()"
---
rerobots-launch 
Launch instance from specified workspace deployment or type

USAGE:
    rerobots launch [OPTIONS] <ID>

FLAGS:
    -h, --help       Prints help information
    -V, --version    Prints version information

OPTIONS:
        --public-key <FILE>    path of public key to use; if not given, then a
                               new key pair will be generated

ARGS:
    <ID>    workspace type or deployment ID

